### PR TITLE
Added Datadog 101 and cleaned up css around videos

### PR DESCRIPTION
**Pull request by technovangelist**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -143,7 +138,6 @@ DEPENDENCIES
   rouge
   sassc
   therubyracer
-  video_info
```
Comment (irabinovitch) [inline Gemfile.lock]: These changes look unrelated?

File: Gemfile [hunk 2]
```diff
@@ -9,7 +9,6 @@ gem 'builder'
 gem 'dogapi', '~> 1.22.0'
 gem 'rouge'
 gem 'oj'
-gem 'video_info'
```
Comment (irabinovitch) [inline Gemfile]: Intentional?

**Comment by irabinovitch** [comment]
Can we push this up to staging first?

File: Gemfile.lock (hunk 1 above)
Comment (technovangelist) [inline Gemfile.lock]: video info was a plugin that made it easier to get the still image for the video. api changed and no longer needed  a long hash that was different from video id. so video_info no longer needed. removed it.

File: Gemfile (hunk 2 above)
Comment (technovangelist) [inline Gemfile]: yes
